phh-on-boot.sh

```diff
@@ -13,3 +13,9 @@ getprop | \
 if grep -qF android.hardware.boot /vendor/manifest.xml;then
 	bootctl mark-boot-successful
 fi
+
+if [ $FOUND_HUAWEI =  1];then
```

Comment:
I don't even know if that's supposed to work.

Reply:
Also, this has to be a lot more granular. 
Ex. Huawei ships RNE without NFC, that, it's one of the very few devices on HI6250 without NFC.

Reply:
Yeah that won't work, but I don't care I just want th idea.
Is there a property we can use on huawei devices to know whether there is NFC?